### DISCUSSION TITLE
Barrows Tomb Door highlighter

### DISCUSSION THREAD

**Discussion by githubnoobpls** [runelite/runelite#14954] [Ideas]
In the barrows final maze you have to find the correct door by manually hovering over individual doors.

I think a good QoL update would be similar to how the agility plugin works with a green highlighted area but for the correct barrows door to the chest room.

I am unsure whether this is possible but thought I'd suggest it as I couldn't see any other suggestion and as I'm 800 chests in I would appreciate this if it was included.

Thanks!

**Comment by geheur** [comment]
FWIW, someone made a PR for this at some point https://github.com/runelite/runelite/pull/13352.

**Reply by hansjm10** [reply]
We might need to find out if this would be allowed. Otherwise I think adding this in would be a great addition to Barrows.
